### ISSUE TITLE
When deactivating a triggered profile with a gesture NVDA provides a confusing info and does not deactivate the profile.

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#12877]
### Steps to reproduce:
1. Create a configuration profile.
2. Create a trigger for it and make it so that it is triggered in a given program.
3. Start the chosen program and make sure that profile has been applied to it.
4. From the input gestures dialog assign a keyboard command to activate and deactivate the profile created in point 1.
5. In the program for which you've created a trigger press the assigned gesture and observe what happens.
### Actual behavior:
Each time the gesture is pressed NVDA reports "ProfileName disabled" yet the profile stays enabled.

### Expected behavior:
I'm not sure what would be ideal here but either one of:
- NVDA reports that triggered profile cannot be disabled
- NVDA disables the profile and falls back to the default configuration

seems a reasonable expectation.
#### NVDA installed/portable/running from source:
N/A
#### NVDA version:
From 2-019.2 up to the latest Alpha
#### Windows version:
N/A
#### Name and version of other software in use when reproducing the issue:
Any program for which trigger can be assigned - my testing has been done with Becky! Internet Mail.
#### Other information about your system:
None relevant
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
As noted above - problem exists from 2019.2 upwards since enabling profiles witht gestures has been introduced with this version.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tried but unlikely to be relevant.

**Comment by Adriani90** [comment]
cc: @nvdaes, @CyrilleB79 I guess this is still reproducible with NVDA 2025.3 right?

**Comment by CyrilleB79** [comment]
Yes, still reproducible on 2025.3.1rc1.

It is not clear what the use case would be for a user to deactivate the profile of the current application.
It's worth noting that for test purpose, one can already check "Temporarily disable all triggers" in the Profiles dialog.
